.docker/assets/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:11.5
+
+# This path is matched in docker-compose.yml
```

Comment:
This comment doesn't make sensee. But since you're just setting the workdir, this `Dockerfile` isn't even needed at all (just set the `working_dir` in `docker-compose.yml` with the `node:11.5` image)

Reply:
I think I just meant that the workdir has to match the volume's directory.